--- a/src/pages/Home.tsx
+++ b/src/pages/Home.tsx
@@ -1,6 +1,7 @@
 import { Clock } from '../components/Clock'
 import { Counter } from '../components/Counter'
 import { Card } from '../components/Card'
+import { WeeklyVisitors } from '../components/graphs/WeeklyVisitors'
 
 export function Home() {
   return (
@@ -14,6 +15,7 @@
       </Card>
       <Clock />
       <Counter />
+      <WeeklyVisitors />
     </div>
   )
 }
